### PR TITLE
Make azure-spring-boot-test-keyvault support both spring-web and spring-webflux.

### PR DESCRIPTION
**Pull request by chenrujun**
- In azure-spring-boot-test-keyvault, add 2 profiles: spring-web, spring-webflux.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-test-aad-b2c/pom.xml [hunk 1]
```diff
@@ -25,28 +26,29 @@
...
     </dependency>
-
     <dependency>
       <groupId>org.thymeleaf.extras</groupId>
       <artifactId>thymeleaf-extras-springsecurity5</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+    </dependency>
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-test-aad-b2c/pom.xml:44]: This dependency is included by `<artifactId>azure-spring-boot-test-selenium-common</artifactId>`

File: eng/versioning/version_client.txt [hunk 2]
```diff
@@ -151,6 +151,8 @@ com.azure.spring:azure-spring-boot-test-aad-resource-server-by-filter;1.0.0;1.0.
 com.azure.spring:azure-spring-boot-test-core;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-keyvault;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-keyvault-reactive;1.0.0;1.0.0
```
Comment (saragluna) [inline eng/versioning/version_client.txt:154]: will the version scripts take care of `pom-reactive.xml`

**Review by saragluna — CHANGES_REQUESTED** [review]

File: eng/versioning/version_client.txt (hunk 2 above)
Comment (chenrujun) [reply eng/versioning/version_client.txt:154]: Good question. 👍 
Now it won't. I'll add a commit to fix it.

File: sdk/spring/azure-spring-boot-test-aad-b2c/pom.xml (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-test-aad-b2c/pom.xml:44]: OK, I'll delete it.

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-test-aad-b2c/pom.xml]: Deleted

File: eng/versioning/version_client.txt (hunk 2 above)
Comment (chenrujun) [reply eng/versioning/version_client.txt:154]: Added.

**Comment by chenrujun** [comment]
Hi, @saragluna . Please help to review this PR. 🙏

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**